pkg/plugins/resources/awsami/condition_test.go

```diff
@@ -21,18 +20,17 @@ func TestCondition(t *testing.T) {
 		d.ami.apiClient = mockDescribeImagesOutput{
 			Resp: d.mockedResponse,
 		}
-		gotResult := result.Condition{}
-		err := d.ami.Condition("", nil, &gotResult)
+		// gotResult := result.Condition{}
```

Comment:
```suggestion
```

Shouldn't this comment be removed?

Reply:
absolutely, looks like I forgot to remove it. Thanks!